core/api/src/graphql/public/types/object/user.ts

```diff
@@ -124,6 +126,13 @@ const GraphQLUser = GT.Object<User, GraphQLPublicContextAuth>({
       },
     },
 
+    supportChat: {
+      type: GT.NonNullList(SupportMessage),
+      resolve: async (source, args, { domainAccount }) => {
+        return getSupportChatMessages(domainAccount.id)
```

Comment:
dont we want to use the tracing wrapper?

Reply:
we should, I will add that.